Gemfile

```diff
@@ -51,7 +51,7 @@ gem "tzinfo-data"
 gem "blueprinter"
 
 gem "govuk-components", ">= 1.1.3"
-gem "govuk_design_system_formbuilder", "~> 2.1", ">= 2.1.5"
+gem 'govuk_design_system_formbuilder', '~> 2.3.0b1'
```

Comment:
What's this? A non-rubocop-compliant edge release? Did someone get their PR with data attributes merged today?